superdesk/internal_destinations.py

```diff
@@ -63,8 +63,8 @@ def handle_item_published(sender, item, desk=None, **extra):
                 continue
             if not filters_service.does_match(content_filter, item):
                 continue
-        if not extra.get("after_scheduled") and dest.get("send_after_schedule"):
-            # if after_schedule is set to False and send_after_schedule is True
+        if not extra.get("after_scheduled") and item[PUBLISH_SCHEDULE] is not None and dest.get("send_after_schedule"):
+            # if after_schedule is set to False and item[PUBLISH_SCHEDULE] is not None and send_after_schedule is True
```

Comment:
would be good to describe it in a more user friendly way, it seems pretty much like the code above

Reply:
yep